### PR TITLE
Add new Tier Three product into product catalog

### PR DESCRIPTION
**Pull request by rupertbates**
## What does this change?
We are changing the way we represent the new tier three product in Zuora as described here: https://docs.google.com/document/d/1NOata7_3YNsZ97bd312QQ9MYJrmko55CaO4D77ExUhI/edit

This PR updates the generate-product-catalog lambda to add this new product into the product catalog.

### CODE REVIEW

File: modules/product-catalog/src/zuoraToProductNameMappings.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const zuoraCatalogToProductKey: Record<string, string> = {
 	'Newspaper Digital Voucher': 'SubscriptionCard',
 	Contributor: 'Contribution',
 	'Newspaper Delivery': 'HomeDelivery',
+	'Tier Three': 'TierThree',
```
Comment (jamesgorrie) [inline modules/product-catalog/src/zuoraToProductNameMappings.ts:12]: I am still a little wary of this - are the business using any other language like "Premium"?

Comment (rupertbates) [reply modules/product-catalog/src/zuoraToProductNameMappings.ts:12]: Premium tier has been mentioned once, but that is has been used in the apps previously so could be confusing. T3 seems to be staying the distance, but I agree it isn't ideal.

Comment (rupertbates) [reply modules/product-catalog/src/zuoraToProductNameMappings.ts:12]: Apparently the business name for it is print + digital, but that clashes with the old print + digital newspaper product. We could go for print + digital 2024 or V2 or something but I don't really like those too much either!

Comment (jamesgorrie) [reply modules/product-catalog/src/zuoraToProductNameMappings.ts:12]: Yea, maybe something that is vague and can change is alright.

**Review by jamesgorrie — APPROVED** [review]
Lovely

**Pull request merged**